cluster-up/cluster/kind/common.sh

```diff
@@ -168,8 +168,17 @@ function setup_kind() {
         done
     fi
 
+    local CNI_VERSION="v0.8.5"
```

Comment:
It will be nicer to move all this section to a function.

Reply:
on it